test/test-fs.c

```diff
@@ -2180,7 +2198,7 @@ TEST_IMPL(fs_utime) {
   check_utime(path, atime, mtime);
   uv_fs_req_cleanup(&req);
 
-  atime = mtime = 1291404900; /* 2010-12-03 20:35:00 - mees <3 */
+  atime = mtime = 1291404900; /* 2010-12-03 20:35:00 */
```

Comment:
Don't remove that!  Neither on line 2295, please. :-)

Reply:
@bnoordhuis just out of curiosity, what is the significance of it?

Reply:
That's when my daughter Mees was born. :-)